packages/lib/server-only/document/seal-document.ts

```diff
@@ -103,11 +104,13 @@ export const sealDocument = async ({
   doc.getForm().flatten();
   flattenAnnotations(doc);
 
-  const certificatePages = await doc.copyPages(certificate, certificate.getPageIndices());
+  if (certificate) {
+    const certificatePages = await doc.copyPages(certificate, certificate.getPageIndices());
 
-  certificatePages.forEach((page) => {
-    doc.addPage(page);
-  });
+    certificatePages.forEach((page) => {
+      doc.addPage(page);
+    });
+  }
```

Comment:
It's not compulsory but it is desirable, this is a stopgap solution since having a sealed document and downloading the certificate later isn't a blocker.

Shortly as we integrate trigger this won't be an issue anymore.